skimage/filters/rank/__init__.py

```diff
@@ -12,7 +12,6 @@
 
 __all__ = ['autolevel',
            'autolevel_percentile',
-           'bottomhat',
```

Comment:
should these be removed now?

Reply:
I removed them to facilitate the tests (it uses this variable)I can handle it differently if necessary

Reply:
__all__ helps with autocomplete and introspection. So I would prefer to keep it until we really don't want users to use it.

Reply:
Hum, I agree here, if it's in the API (albeit deprecated, it should stay in `__all__`).